source/vstd/slice.rs

```diff
@@ -10,18 +10,30 @@ pub use super::std_specs::vec::VecAdditionalSpecFns;
 
 verus! {
 
-pub trait SliceAdditionalSpecFns<T> {
+impl<T> View for [T] {
+    type V = Seq<T>;
+
     spec fn view(&self) -> Seq<T>;
+}
+
+impl<T: DeepView> DeepView for [T] {
+    type V = Seq<T::V>;
+
+    open spec fn deep_view(&self) -> Seq<T::V> {
+        let v = self.view();
+        Seq::new(v.len(), |i: int| v[i].deep_view())
+    }
+}
 
+#[verusfmt::skip] // this doesn't help
```

Comment:
Regarding this line: the `#[verusfmt::skip]` attribute doesn't work if a file is not parseable by verusfmt; I've opened an issue to track supporting full-file skips at https://github.com/verus-lang/verusfmt/issues/59 

As for the blocker on formatting, that fix (https://github.com/verus-lang/verusfmt/pull/58) has been merged and released. I gave the fmt CI on this PR a kick, and its current complaint is the number of spaces on this line 🙃  If you remove the skip, the formatting succeeds:

```suggestion
```